chain/chunks/src/shards_manager_actor.rs

```diff
@@ -185,6 +189,14 @@ pub(crate) struct ChunkRequestInfo {
     last_requested: time::Instant,
 }
 
+#[derive(Clone, Debug)]
```

Comment:
```suggestion
#[derive(Debug)]
```

I don't think clone is needed.